doc/cephfs/ceph-dokan.rst

```diff
@@ -8,7 +7,9 @@ Mount CephFS on Windows
 It leverages Dokany, a Windows driver that allows implementing filesystems in
 userspace, pretty much like FUSE.
 
-Please check the `installation guide`_ to get started.
+Please see the `installation guide`_ to get started.
+
+Please see the `OS recommendations`_ regarding client support.
```

Comment:
Should this be highlighted as a note?
```suggestion
.. note::

Please see the `OS recommendations`_ regarding client packages support.
```

Reply:
The word "package" hasn't been added here like in two other places.

Also neither this note nor the one in `doc/rbd/rbd-windows.rst` are highlighted as a note (i.e. using `.. note::` syntax).  I'm fine with them not being highlighted, but wanted to raise this again just in case because it looks like this conversation got marked as resolved mistakenly.

Reply:
Addressed in https://github.com/ceph/ceph/pull/64483